projects/go-style-guide/src/app/features/ui-kit/components/button-docs/button-docs.component.ts

```diff
@@ -123,7 +106,7 @@ export class ButtonDocsComponent {
   private buttonTemplate (variant: string, icon: string): string {
```

Comment:
I know this isn't part of your work here, but is there an unnecessary space after this method name?